website/content/en/preview/troubleshooting.md

```diff
@@ -86,3 +86,48 @@ Your security groups are not blocking you from reaching your webhook.
 
 This is especially relevant if you have used `terraform-eks-module` version `>=18` since that version changed its security
 approach, and now it's much more restrictive.
+
+## DaemonSets can result in deployment failures
+
+For Karpenter versions 0.5.3 and earlier, Daemonsets were not properly considered when provisioning nodes.
+This sometimes caused nodes to be deployed that could not meet the needs of the requested Daemonsets.
+The result could be log messages like the following:
+
+```text
+Excluding instance type r3.8xlarge because there are not enough resources for daemons {"commit": "7e79a67", "provisioner": "default"}
+```
```

Comment:
This doesn't sound right to me. The problem for Daemon set before v0.5.3 is that Karpenter doesn't consider the resource requests of daemon sets when selecting instance type. Karpenter will launch a smaller node and pod will fail to run due to not enough resources. 

The above error log happens when Karpenter bin-packing is filtering instance types according to just the Daemon-set resource requests. It is not an indicator of the problem.